Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Widgets/NodePalette/NodePaletteWidget.cpp

```diff
@@ -85,7 +85,7 @@ namespace GraphCanvas
                 if (leftSpot < textRect.right())
                 {
                     int visibleLength = AZStd::GetMin(selectedTextLength, textRect.right() - leftSpot);
-                    QRect highlightRect(textRect.left() + preSelectedTextLength, textRect.top(), visibleLength, textRect.height());
+                    QRect highlightRect(textRect.left() + preSelectedTextLength + 4, textRect.top(), visibleLength, textRect.height());
```

Comment:
nit: would recommend removing "magic numbers" from code and instead having a constant (either a const in this function, or a static const in the class).